kafka-ui-api/src/test/java/com/provectus/kafka/ui/emitter/MessageFiltersTest.java

```diff
@@ -21,7 +21,7 @@
 class MessageFiltersTest {
 
   @Nested
-  class StringContainsFilter {
+  class StringContainsFilterTest {
```

Comment:
this naming was intended to have behaviour-like test name like 
`GroovyScriptFilter.throwsExceptionOnInvalidGroovySyntax `